### DISCUSSION TITLE
expect_column_values_to_match_regex doesn't give the expected results.

### DISCUSSION THREAD

**Discussion by LakshmiTeja08** [great-expectations/great_expectations#7468] [Support]
I am trying to create a rule using regex operation of finding values which matches digits with [+-]15.2 decimal places.
Before dot it can have values of 1 to 15 and after dot it should have or 1 or 2 values. It can accept negative values too.
I tried with following regex operation but it doesn't give the expected results. It also includes correct values as error.
Regex used: "^[+-]?(\\d{1,15}+\\.?\\d{2}|\\d{1,15}\\.?\\d{1}|\\d{1,15})$"

Could you please help me if I am using the correct regex?

**Comment by talagluck** [comment]
Hi @LakshmiTeja08 - have you tried using a regex tester, like https://regex101.com/? That would be the best way to confirm that your regex will yield the desired results.

**Comment by JustinGuese** [comment]
same for me! https://github.com/great-expectations/great_expectations/issues/7656
and yes it's really simple regex as well , like ^\d{4}$
